src/main/kotlin/org/cascadebot/cascadebot/utils/placeholders/Placeholder.kt

```diff
@@ -0,0 +1,45 @@
+package org.cascadebot.cascadebot.utils.placeholders
+
+import org.cascadebot.cascadebot.data.language.Language
+import org.cascadebot.cascadebot.data.language.Locale
+
+abstract class Placeholder<T>(val key: String, val absoluteKey: String) {
+
+    val localisedInfo: Map<Locale, PlaceholderInfo> by lazy {
```

Comment:
what does `by lazy` do?

Reply:
it's lazy initialisation, basically runs the code inside the block first time it's run

Reply:
ahh ok.